### ISSUE TITLE
Update Hebrew translations

### ISSUE THREAD

**Issue by dngray** [squidfunk/mkdocs-material#5090]
### Translations

```Jinja
{% macro t(key) %}{{ {
  "language": "he",
  "action.edit": "עריכת הדף הזה",
  "action.skip": "לדלג לתוכן",
  "action.view": "צפה במקור של דף זה ⬅️",
  "announce.dismiss": "לא להציג את זה שוב",
  "blog.archive": "ארכיון ⬅️",
  "blog.categories": "קטגוריות ⬅️",
  "blog.categories.in": "בתוך⬅️",
  "blog.continue": "המשך לקרוא ⬅️",
  "blog.draft": "טיוטה ⬅️",
  "blog.index": "חזרה לאינדקס ⬅️",
  "blog.meta": "מטא-נתונים ⬅️",
  "blog.references": "קישורים קשורים ⬅️",
  "clipboard.copy": "העתקה ללוח",
  "clipboard.copied": "הועתק ללוח",
  "consent.accept": "לקבל",
  "consent.manage": "לנהל הגדרות",
  "consent.reject": "לדחות",
  "footer": "כותרת תחתונה",
  "footer.next": "הבא",
  "footer.previous": "הקודם",
  "header": "כותרת עליונה",
  "meta.comments": "הערות",
  "meta.source": "מקור",
  "nav": "ניווט",
  "readtime.one": "קריאה 1 דקות ⬅️",
  "readtime.other": "# דקות קריאה ⬅️",
  "rss.created": "RSS הזנת ⬅️",
  "rss.updated": "הזנת RSS של תוכן מעודכן ⬅️",
  "search": "חיפוש",
  "search.placeholder": "חיפוש",
  "search.share": "שיתוף",
  "search.reset": "ניקוי",
  "search.result.initializer": "אתחול חיפוש",
  "search.result.placeholder": "יש להקליד כדי להתחיל לחפש",
  "search.result.none": "אין מסמכים תואמים",
  "search.result.one": " מסמך1 תואם",
  "search.result.other": "# מסמך תואם",
  "search.result.more.one": "עוד אחד בדף הזה",
  "search.result.more.other": "עוד # בדף הזה",
  "search.result.term.missing": "חסר",
  "select.language": "בחירת שפה",
  "select.version": "בחירת גרסה",
  "source": "לעבור אל המאגר",
  "source.file.contributors": "תורמים ⬅️",
  "source.file.date.created": "נוצר",
  "source.file.date.updated": "עדכון אחרון",
  "tabs": "לשוניות",
  "toc": "תוכן העניינים",
  "top": "חזרה למעלה"
}[key] }}{% endmacro %}
```

### Before submitting

- [X] I've checked the list of [available languages](https://squidfunk.github.io/mkdocs-material/setup/changing-the-language/#site-language) for existing translations.
- [X] I assure that the translations are accurate to my best knowledge.
- [ ] __Optional__: I want to integrate this translation myself and create a pull request following the [contribution guide](https://github.com/squidfunk/mkdocs-material/blob/master/CONTRIBUTING.md).

**Comment by dngray** [comment]
This is from our translation team.

**Comment by squidfunk** [comment]
Thanks, perfect! Added in 8f4d30f8f.

**Comment by squidfunk** [comment]
Released as part of [9.0.15](https://github.com/squidfunk/mkdocs-material/releases/tag/9.0.15).
